daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -19,8 +19,8 @@ FAIL=0
 FAILURES=""
 
 function wait_for_connectivity {
-  if [[ $(cat /etc/centos-release) =~ "CentOS release 6" ]]; then
-    status "CentOS 6: Waiting for network connectivity."
+  if [[ $(cat /etc/centos-release) =~ "CentOS release 6" && $(command -v nmcli) ]]; then
```

Comment:
useless use of cat :)
`if grep -q 'CentOS release 6' /etc/centos-release && command -v nmcli; then`

Reply:
nice! Updated and re-ran tests.